projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cds/merchandising-carousel.ts

```diff
@@ -344,12 +344,13 @@ export function navigateToHomepage(): void {
 
 export function navigateToCategory(
   categoryName: string,
-  categoryCode: string
+  categoryCode: string,
+  force: boolean = true
 ): void {
   const categoryPage = waitForCategoryPage(categoryCode, 'getCategory');
   cy.get('cx-category-navigation cx-generic-link a')
     .contains(categoryName)
-    .click({ force: true });
+    .click({ force });
```

Comment:
Was this test flaky?

Reply:
I'm not sure if it was flaky, but it seemed like bad practice to always force the click. That leads to clicks occurring while the ui is not technically [actionable](https://docs.cypress.io/guides/core-concepts/interacting-with-elements#Actionability). In my opinion, if a test is written correctly, force should never be necessary. So I made it optional to force the click (though force is still true by default to avoid breaking changes).